### PR TITLE
FIO | integrate with internal ES server

### PR DESCRIPTION
**Pull request by Avilir**
Integrating trhe Internal elasticsearch server in this test so it can be run on every platform, including the public cloud

### CODE REVIEW

File: tests/e2e/performance/test_fio_benchmark.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 """
 import logging
 import pytest
-from elasticsearch import Elasticsearch
+from elasticsearch import (Elasticsearch, exceptions as ESExp)
```
Comment (RazTamir) [inline tests/e2e/performance/test_fio_benchmark.py:6]: no need for parentheses

File: tests/e2e/performance/test_fio_benchmark.py [hunk 2]
```diff
@@ -12,14 +13,25 @@
 from ocs_ci.ocs.utils import get_pod_name_by_pattern
 from ocs_ci.ocs.ripsaw import RipSaw
 from ocs_ci.ocs import constants
-from ocs_ci.framework.testlib import (
-    E2ETest, performance, rh_internal_lab_required
-)
+from ocs_ci.framework.testlib import (E2ETest, performance)
```
Comment (RazTamir) [inline tests/e2e/performance/test_fio_benchmark.py:16]: same

File: tests/e2e/performance/test_fio_benchmark.py [hunk 3]
```diff
@@ -12,14 +13,25 @@
...
 log = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope='function')
+def es(request):
+    def teardown():
+        es.cleanup()
+
+    request.addfinalizer(teardown)
+
```
Comment (RazTamir) [inline tests/e2e/performance/test_fio_benchmark.py:29]: remove empty line

File: tests/e2e/performance/test_fio_benchmark.py (hunk 1 above)
Comment (Avilir) [reply tests/e2e/performance/test_fio_benchmark.py]: done.

File: tests/e2e/performance/test_fio_benchmark.py (hunk 2 above)
Comment (Avilir) [reply tests/e2e/performance/test_fio_benchmark.py]: done

File: tests/e2e/performance/test_fio_benchmark.py (hunk 3 above)
Comment (Avilir) [reply tests/e2e/performance/test_fio_benchmark.py]: done

**Pull request closed without merging**